code/modules/modular_computers/file_system/reports/crew_record.dm

```diff
@@ -132,12 +131,26 @@ GLOBAL_VAR_INIT(arrest_security_status, "Arrest")
 	// Antag record
 	set_antagRecord(H?.exploit_record && !jobban_isbanned(H, "Records") ? html_decode(H.exploit_record) : "")
 
+// Simple record to HTML (for paper purposes) conversion.
+// Not visually that nice, but it gets the work done, feel free to tweak it visually
+/datum/computer_file/report/crew_record/proc/record_to_html(access)
+	var/dat = "<tt><H2>RECORD DATABASE DATA DUMP</H2><i>Generated on: [stationdate2text()] [station_time_timestamp("hh:mm")]</i><br>******************************<br>"
+	dat += "<table>"
+	for(var/datum/report_field/F in src.fields)
+		if(F.verify_access(access))
+			dat += "<tr><td><b>[F.display_name()]</b>"
+			if(F.needs_big_box)
+				dat += "<tr>"
+			dat += "<td>[F.get_value()]"
+	dat += "</tt>"
+	return dat
+
 // Global methods
 // Used by character creation to create a record for new arrivals.
-/proc/CreateModularRecord(mob/living/carbon/human/H)
+/mob/proc/CreateModularRecord()
```

Comment:
move above //Global methods comment since this is now a local proc

Reply:
done (nerd)